src/ariadne.c

```diff
@@ -224,6 +224,10 @@ static long triangle_findSE8(long ptfind_x, long ptfind_y);
 long ma_triangle_route(long ptfind_x, long ptfind_y, long *ptstart_x);
 void edgelen_init(void);
 /******************************************************************************/
+
+// remove these pragma directives above and below the ariadne_compare_ways function, when ariadne_compare_ways is used by KFX code
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
 static void ariadne_compare_ways(const struct Ariadne *arid1, const struct Ariadne *arid2)
```

Comment:
You could have just removed the static.
But a comment is still appropriate so people know why it's unused.

Reply:
thanks, good to know.

I'll change this soon.